### PR TITLE
Add a background img, change selection colors and other

### PR DESCRIPTION
**Pull request by codingoak**

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -3,15 +3,26 @@
...
 }
 
+::-moz-selection {
+  color: lightblue; background: rebeccapurple;
+}
+
+::selection {
+  color: lightblue; background: rebeccapurple;
+}
+
```
Comment (JelenadlS) [inline styles.css:25]: Hey Daniel, may I ask for what do you use it? I do not really get it even by checking google (just interested) :-)

File: styles.css [hunk 2]
```diff
@@ -35,13 +46,13 @@ h1, h2, h3 {
...
-  font-family: 'Open Sans', sans-serif;
   max-width: 800px;
   margin: 1rem auto;
   padding: 0 0.5rem;
   border: 1px solid rgba(0, 0, 0, 20%);
-  border-radius: 1rem;
-  background: linear-gradient(90deg, rgba(0, 138, 138, 3%), rgba(0, 138, 138, 6%), rgba(0, 138, 138, 3%));
+  border-radius: 0.5rem;
+  background-color: rgba(103, 175, 204, 95%);
+
```
Comment (JelenadlS) [inline styles.css:55]: I think this empty row can be deleted.

**Review by JelenadlS — CHANGES_REQUESTED** [review]
I think it looks really good - I am impressed :-)
I was not able to add a comment as it was already merged/ approved however, i think the <div> in HTML row 74 has no closing tag.

File: styles.css (hunk 2 above)
Comment (codingoak) [reply styles.css:55]: yes this line was not necessary - thx

File: styles.css (hunk 1 above)
Comment (codingoak) [reply styles.css:25]: `::-moz-selection {
  color: lightblue; background: rebeccapurple;
}

::selection {
  color: lightblue; background: rebeccapurple;
}`

this if found to change the color for selected text and the background color

**Pull request merged**